tools/sidekick.js

```diff
@@ -0,0 +1,351 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+/* global window, document, HTMLElement */
+
+'use strict';
+
+ /* this file moved to: https://www.project-helix.io/bookmarklets/sidekick.js */
```

Comment:
If I understand it well, the generic code will be moved to project-helix.io when merged to master there. Will be a lot of less code.

Reply:
Correct: only /tools/sidekick/plugins.js will remain here.